src/Telemetry/Library/TelemetryManagerConfig.cs

```diff
@@ -39,13 +55,30 @@ public string EventNamePrefix
             set { this.eventNamePrefix = ValidateName(value); }
         }
 
+        /// <summary>
+        /// The default User Consent to collect telemetry.
+        /// If the default value is ConsentKind.OptedOut, then the user can opt-in via
+        /// the environment variable defined in TelemetryOptInVariableName.
+        /// If the default value is ConsentKind.OptedIn, then the user can opt-out via
+        /// the environment variable defined in TelemetryOptOutVariableName.
+        /// </summary>
+        public ConsentKind DefaultTelemetryConsent { get; set; } = ConsentKind.OptedOut;
+
         /// <summary>
         /// The name of the environment variable to be checked at initialization to see if
         /// the user wants to opt-out of sending telemetry data.
         /// If the value of the environment variable is "1", no data will be collected or sent.
         /// </summary>
         public string TelemetryOptOutVariableName { get; set; } = "QDK_TELEMETRY_OPT_OUT";
```

Comment:
I'm unsure why we would need both the TelemetryOptOutVariableName as well as the TelemetryOptInVariableName?

Reply:
Hi Scott, that's a great point but I was trying to keep consistency with how we handle telemetry opt-out for other products.
In general, either the default consent option will be opted-in and TelemetryOptOutVariableName can be used to turn it off, or the default consent option will be opted-out and TelemetryOptInVariableName can be used to to turn it on.